packages/neo4j-driver/test/driver.test.js

```diff
@@ -588,6 +588,20 @@ describe('#integration driver', () => {
     })
   })
 
+  it('hasReachableServer success', async () => {
+    const result = await neo4j.hasReachableServer(`${sharedNeo4j.scheme}://${sharedNeo4j.hostname}`)
+    expect(result).toBe(true)
+  })
+
+  it('hasReachableServer failure', async () => {
+    try {
+      await neo4j.hasReachableServer(`${sharedNeo4j.scheme}://${sharedNeo4j.hostname}:9999`)
+      expect(true).toBe('false')
+    } catch (error) {
+      expect(error).toBeInstanceOf(Error)
+    }
```

Comment:
Doesn't that catch the assertion error should the assertion fail?

Reply:
Good catch. I fixed the test to make it correct.